manifests/3.0.0/opensearch-3.0.0-test.yml

```diff
@@ -4,7 +4,7 @@ name: OpenSearch
 ci:
   image:
     name: opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1
```

Comment:
@peterzhuamazon what would be the correct (latest?) CI runner image?

Reply:
You dont need to change this as we will always link the latest minor version to major version.

ex: ci-runner-al2-opensearch-build-v1 will always point to latest which is ci-runner-al2-opensearch-build-v1.4 that I just built yesterday.